AppCenterReactNativeShared/Products/AppCenterReactNativeShared/LICENSE.md

```diff
@@ -0,0 +1,13 @@
+Visual Studio App Center Plugin for React Native
```

Comment:
We don't need to add this file because it's automatically copied as part of the SDK release process at [here](https://github.com/Microsoft/AppCenter-SDK-React-Native/blob/develop/AppCenterReactNativeShared/zip-framework.sh#L11).

Reply:
I agree with modifying build script, but without this file ```react-native link``` outputs annoying error when building our TestApp and with cocoapods 1.4 installed.